treeopt/visualize.py

```diff
@@ -0,0 +1,370 @@
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import matplotlib as mpl
+import numpy as np
+
+
+class Visualize:
+    def __init__(self, optimizer):
+        """
+        Initializes a visiualize Class
+        :param optimizer: Object of the TreeOpt Class containing all
+        information of the calculated metamodell
+        :type optimizer: TreeOpt Class Object
+        :return: Nothing
+        :rtype: None
+        """
+
+        self.opti_data = optimizer
+        self.num_ele = 100
+
+    def set_n(self, n):
+        """
+        Sets the number of points which on which a metamodell is to be
+        evaluated alongside each axis, in order to generate a plot
+        :param n: Number of Evaluations alongside each axis (default n=100)
+        :type n: Integer
+        :return: Nothing
+        :rtype: None
+        """
+        self.num_ele = n
+
+    def plot_benchmark_2vars(self):
+        """
```

Comment:
What if I throw at this a problem with more than two vars? Do I get an error? Can I choose two variables of my higher dimensional problem, that I get visualized?

Reply:
there is the plot function, which calls this function if a specific keyword is passed. At the moment I cannot choose two variables of a higher dimensional problem.